crates/hyperswitch_domain_models/src/payments.rs

```diff
@@ -509,6 +509,8 @@ where
 {
     pub flow: PhantomData<F>,
     pub payment_intent: PaymentIntent,
+    pub email: Option<pii::Email>,
```

Comment:
which email is this? This can be taken from the customer object if related to customer right?